File: Python Solutions/1012.py
dados= input().split()
a= float(dados[0])
b= float(dados[1])
c= float(dados[2])

triangulo= a*c/2
circulo= 3.14159 * c**2
trapezio= ((a+b)*c)/2
quadrado= b**2
reatangulo= a*b

print(f"TRIANGULO: {triangulo:.3f}")
print(f"CIRCULO: {circulo:.3f}")
print(f"TRAPEZIO: {trapezio:.3f}")
print(f"QUADRADO: {quadrado:.3f}")
print(f"RETANGULO: {reatangulo:.3f}")

File: JavaScript Solutions/1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

let A = parseFloat(lines[0]),
    B = parseFloat(lines[1]),
    C = parseFloat(lines[2]),
    triangle = (A*C/2).toFixed(3),
    circle = (3.14159 * C*C).toFixed(3),
    trapezium = ((A+B)/2*C).toFixed(3),
    square = (B*B).toFixed(3),
    rectangle = (A*B).toFixed(3);
    
console.log("TRIANGULO: "+triangle);    
console.log("CIRCULO: "+circle);    
console.log("TRAPEZIO: "+trapezium);    
console.log("QUADRADO: "+square);    
console.log("RETANGULO: "+rectangle);
